nixos/modules/config/sysfs.nix

```diff
@@ -0,0 +1,192 @@
+{
+  lib,
+  config,
+  ...
+}:
+
+let
+  inherit (builtins)
```

Comment:
All those can be directly taken from lib. Also the lib variants sometimes have fallbacks for older nix versions.

Reply:
I’ll change it and keep it in mind for the future.